index.js

```
@@ -0,0 +1,93 @@
+    console.error('OpenAI error:', err.message);
+    res.status(500).json({ error: 'Failed to process message' });
+  }
+});
+
+// WebSocket for A2A direct interactions
+const wss = new WebSocket.Server({ noServer: true });
+wss.on('connection', ws => {
+  ws.on('message', async data => {
+    const { to, message } = JSON.parse(data);
+    const toAgent = agents.get(to);
+    if (toAgent && toAgent.ws) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)